drivers/include/periph/uart.h

```diff
@@ -137,6 +172,21 @@ enum {
  */
 int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg);
 
+/**
+ * @brief   Setup parity, data and stop bits for a given UART device
+ *
+ * @param[in] uart          UART device to initialize
+ * @param[in] data_bits     number of data bits in a UART frame
+ * @param[in] parity        parity mode
+ * @param[in] stop_bits     number of stop bits in a UART frame
+ *
+ * @return                  UART_OK on success
+ * @return                  UART_NODEV on invalid UART device
```

Comment:
just wanted to ACK, but noticed one more thing (sorry...): the interface description does not match with the implementation above: the `uart_mode()` function should not return the `UART_NODEV` error code, but should use `assert()` internally to check the given device (as done in the implementation above). So please remove the line above.